cmd/admin-config-get.go

```diff
@@ -59,10 +60,12 @@ func (u configGetMessage) String() string {
 // JSON jsonified service status Message message.
 func (u configGetMessage) JSON() string {
 	u.Status = "success"
-	statusJSONBytes, e := json.Marshal(u)
+	statusJSONBytes, e := json.MarshalIndent(u, "", "    ")
 	fatalIf(probe.NewError(e), "Unable to marshal into JSON.")
 
-	return string(statusJSONBytes)
+	//Remove \n and \t from config.json string
```

Comment:
It is not a big deal, but it could be confusing to call `MarshalIndent` above to introduce indentations and new lines and then remove the same control characters here. 
I guess, it would be clearer if we make the distinction between these 2 cases by adding comments, that we are removing the tabs and new line control characters from the internal config fields from `u.Config` at sub-level, not `config.json` file which doesn't exist anymore.

Reply:
done